### ISSUE TITLE
Add support for units of measurement

### ISSUE THREAD

**Issue by joroque** [joroque/magicseaweed#1]
Magicseaweed API returns the forecast in UK units by default: _ft_, _mph_ and _c_. Although, it is possible to specify alternative units by passing an URL query option.

```
http://magicseaweed.com/api/YOURAPIKEY/forecast/?spot_id=10&units=eu
```

That will return _m_, _kph_ and _c_.

Supported options are:
- UK (ft, mph, c)
- US (ft, mph, f)
- EU (m, kph, c)

**Comment by gsquire** [comment]
@romeroqj I submitted a pull request with this feature.

**Comment by joroque** [comment]
@gsquire Thanks a lot! I added tests and updated README with the usage of this feature.

Closed by f783e6fe90e68629e5739e5e163c6ad69f19f2b2
